src/dbt/kipptaf/models/students/intermediate/int_extracts__student_enrollments.sql

```diff
@@ -195,6 +195,13 @@ select
         false
     ) as met_fafsa_requirement,
 
+    if(
+        -- hardcode year because PAT wasnt incorporated until SY2025-26
+        e.region = 'Paterson' and e.academic_year <= 2024,
+        e.prevstudentid,
+        e.student_number
+    ) as student_number_historic,
```

Comment:
this is already covered by #3168 as `prevstudentid`

Reply:
can you add the year to the upstream one? I don't see any other use for this anyway. might even want to rename it something like `pearson_local_student_identifier`